src/components/Home/LandingHero/index.tsx

```diff
@@ -48,6 +52,22 @@ const HeroButtons = () => (
       onClick={logUseCasesEvent}
       href="#use-cases"
     />
+    <TwoRowsButtonLink
+      mode="azure"
+      className={cn(styles.actionButton, styles.watchVideo)}
+      title="DVC For VS Code"
+      description="Get VS Code Extension"
```

Comment:
Other options:

<img width="1071" alt="Screenshot 2023-07-19 at 10 37 10 AM" src="https://github.com/iterative/dvc.org/assets/43496356/9abd1687-484c-4cf2-85ae-f676da947764">
<img width="1077" alt="Screenshot 2023-07-19 at 10 36 44 AM" src="https://github.com/iterative/dvc.org/assets/43496356/e54027fa-4756-4314-b06a-3ef07f2cf50a">

Reply:
Text could also be improved 🤔 . I was having difficulty choosing text that wasn't lengthy.

Reply:
Personally, I would be in favour of having just 2 action items. 

What do you think about linking it with text with the proper highlight (bold/colour/icons)?

Example:
<img width="512" alt="Screenshot 2023-07-19 at 22 44 26" src="https://github.com/iterative/dvc.org/assets/20840228/945751d8-a123-4813-a162-eee0bed93708">

Reply:
I like the last one. I would may be make the button the color of VS Code + put VS Code logo there

Reply:
> I like the last one. I would may be make the button the color of VS Code + put VS Code logo there

I was avoiding using the logo since it's technically against VS Code guidelines but we do use it in our product dropdown anyway :) Here's what that looks like:

<img width="1111" alt="image" src="https://github.com/iterative/dvc.org/assets/43496356/f61da7d6-dba6-4bb8-b8dc-e74fa33b3fdb">

Personally, I think the VSCode colors look out of place. Here's how the button looks with blue or clear theme colors:

<img width="1088" alt="image" src="https://github.com/iterative/dvc.org/assets/43496356/f09b86ea-dbba-47bf-8211-0eea76d76b08">

<img width="1093" alt="image" src="https://github.com/iterative/dvc.org/assets/43496356/6585030a-88dd-44fc-a254-23ae82363b8e">

Reply:
> Personally, I think the VSCode colors look out of place. Here's how the button looks with blue or clear theme colors:

it's totally fine :). Let's go with it please.